internal/test/e2e/cluster_test.go

```diff
@@ -183,31 +183,32 @@ func TestNewTestCluster(t *testing.T) {
 	defer cluster.Shutdown()
 	tt := cluster.tt
 
-	// See if autopilot is running by triggering the loop.
-	_, err := cluster.Autopilot.Trigger(false)
-	tt.OK(err)
-
-	// Add a host.
+	// add a host & wait for contracts to form
 	cluster.AddHosts(1)
-
-	// Wait for contracts to form.
-	var contract api.ContractMetadata
 	contracts := cluster.WaitForContracts()
-	contract = contracts[0]
+	if len(contracts) != 1 {
+		t.Fatal("expected 1 contract, got", len(contracts))
+	}
+	contract := contracts[0]
+
+	// fetch autopilot state
+	as, err := cluster.Bus.AutopilotState(context.Background())
```

Comment:
Alternatively to this we could also split this up into 2 routes. 
1. `AutopilotConfig` which returns the config
2. `AutopilotPeriod` which returns the current period

Reply:
Hm yeah, considered that too when writing it but went this direction because we update separately but always need both of the. I split it up though, more symmetrical that way.